ADOPTERS.md

```diff
@@ -0,0 +1,14 @@
+# Octant Adopters
+
+## Success Stories
+
+Below is a list of adopters of Octant in **production environments** that have
```

Comment:
Octant's target and focus is at developers. That said, you don't really "deploy" Octant so "in" seems like an odd term to use.

Reply:
We do want to know if people do use Octant to verify/troubleshoot etc their production environments, right?